### PR TITLE
Document InertiaLink headers option

### PR DESCRIPTION
**Pull request by RobertBoes**
Adds documentation for inertiajs/inertia#204

Took the text from https://inertiajs.com/requests#custom-headers because it's the same feature

### CODE REVIEW

File: pages/links.mdx [hunk 1]
```diff
@@ -110,6 +110,36 @@ You can add data using the `data` attribute. This can be an `object`, or a `Form
...
+      language: 'twig',
+      code: dedent`
+        <inertia-link href="/endpoint" :headers="{ foo: bar }">Save</inertia-link>
+      `,
+    },
+    {
+      name: 'React',
+      language: 'html',
+      code: dedent`
+        <InertiaLink href="/endpoint" headers={{ foo: bar}}>Save</InertiaLink>
```
Comment (reinink) [inline pages/links.mdx:130]: Missing space after `bar`.

File: pages/links.mdx [hunk 2]
```diff
@@ -110,6 +110,36 @@ You can add data using the `data` attribute. This can be an `object`, or a `Form
...
+      `,
+    },
+    {
+      name: 'Svelte',
+      language: 'html',
+      code: dedent`
+        <InertiaLink href="/endpoint" headers={{ foo: bar}}>Save</InertiaLink>
```
Comment (reinink) [inline pages/links.mdx:137]: And again, missing space after `bar`.

**Comment by reinink** [comment]
Awesome, thanks @RobertBoes! Two small corrections needed. 👍

**Comment by RobertBoes** [comment]
Didn't notice the missing spaces, since I copied the code from the `data` examples. Also updated those examples.

**Comment by reinink** [comment]
Heh, awesome, thanks! 👍

**Pull request merged**